tests/testthat/test-test.R

```diff
@@ -0,0 +1,9 @@
+test_that("check_edition() validates inputs", {
+  expect_error(check_edition(20), "not available")
+  expect_error(check_edition("x"), "single number")
+  expect_equal(check_edition(1.5), 1)
+
+  expect_equal(check_edition(), packageVersion("testthat")[[1, 1]])
+  expect_equal(2 * 2, 4)
```

Comment:
What's this about?

Reply:
Oops, accidentally forgot to delete part of the tempalte